lib/util.js

```diff
@@ -383,6 +383,14 @@ const util = {
     }
   },
 
+  collectObjs: (options = config.defaultOptions) => {
+    console.log('collecting object files ...')
```

Comment:
nit: How about moving this log under the below if clause?
I think this log only has meaning when collecting will be started.

Reply:
we removed that part at all